package.json

```diff
@@ -61,7 +61,7 @@
     "audit:public": "npm audit --registry https://registry.npmjs.org/"
   },
   "dependencies": {
-    "@brightside/imperative": "2.7.4",
+    "@brightside/imperative": "^2.7.4",
```

Comment:
Is there a reason the ^ was added to the version number here?

Reply:
I believe it's ok to have it in case someone builds from source and wanted to receive features and fixes without having to reinstall imperative with the corresponding tag every time :) 
I could be wrong though

Reply:
We can change it later if we should be fully locking things down :)